### ISSUE TITLE
PyPi versions & Git tags

### ISSUE THREAD

**Issue by mweinelt** [rhasspy/gruut#13]
PyPi currently has v1.2.3, while on Git the latest tag is v1.2.0

I get notifications from repology to update to v1.2.3, but I can't, since there is no matching version on Git.

This is likely because you rewrote the v1.2.0 tag on Git, and had to move forward with a higher version number on PyPi, because you already published v1.2.2.

This makes packaging slightly more inconvenient than it has to be.

This is also a thing on the gruut-ipa repository, and I somewhat understand with the sheer number of repositories you are maintaining, that this can amount to some non-negligible amount of work.

**Comment by synesthesiam** [comment]
Yeah, I'd say I'm reaching the limits of my process. Any specific advice on branch/tag structure that would make this better for folks like yourself?

**Comment by mweinelt** [comment]
Automating push of pypi packages via github actions on when git tags are pushed? 

And some process that makes sure that the version is kept in sync, like a `VERSION` file that is sourced by the setup.py and also used by a script to prepare the release?

**Comment by mweinelt** [comment]
2.2.2 up on PyPi (https://pypi.org/project/gruut/), Git still on 2.2.0. Pushing a tag would be appreciated, thank you for your work.
